{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -9,96 +9,128 @@ Contributing
 Contributions are welcome, and they are greatly appreciated! Every little bit
 helps, and credit will always be given.
 
-You can contribute in many ways:
-
 Types of Contributions
 ----------------------
 
-Report Bugs
-~~~~~~~~~~~
-
-Report bugs at {{ cookiecutter.git_service }}/{{ cookiecutter.git_username }}/{{ cookiecutter.project_slug }}/issues.
+Report Bugs or Suggest Features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Fix Bugs
-~~~~~~~~
+The best place for this is {{ cookiecutter.git_service }}/{{ cookiecutter.git_username }}/{{ cookiecutter.project_slug }}/issues.
 
-Look through the {% if rwth %}GitLab{% elif github %}GitHub{% endif %} issues for bugs. Anything tagged with "bug" and "help
-wanted" is open to whoever wants to implement it.
+Fix Bugs or Implement Features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Implement Features
-~~~~~~~~~~~~~~~~~~
-
-Look through the {% if rwth %}GitLab{% elif github %}GitHub{% endif %} issues for features. Anything tagged with "enhancement"
-and "help wanted" is open to whoever wants to implement it.
+Look through {{ cookiecutter.git_service }}/{{ cookiecutter.git_username }}/{{ cookiecutter.project_slug }}/issues for bugs or feature request
+and contact us or comment if you are interested in implementing.
 
 Write Documentation
 ~~~~~~~~~~~~~~~~~~~
 
-{{ cookiecutter.project_name }}https://github.com/pyfar/pyfar could always use more documentation, whether as part of the
-official {{ cookiecutter.project_name }} docs, in docstrings, or even on the web in blog posts,
+{{ cookiecutter.project_slug }} could always use more documentation, whether as part of the
+official {{ cookiecutter.project_slug }} docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development.
+Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development using the command-line interface. Note that several alternative user interfaces exist, e.g., the Git GUI, `GitHub Desktop <https://desktop.github.com/>`_, extensions in `Visual Studio Code <https://code.visualstudio.com/>`_ ...
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on {% if rwth %}GitLab{% elif github %}GitHub{% endif %}.
-2. Clone your fork locally::
+1. `Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo/>`_ the `{{ cookiecutter.project_slug }}` repo on GitHub.
+2. Clone your fork locally and cd into the {{ cookiecutter.project_slug }} directory::
 
-    $ git clone {{ cookiecutter.git_service }}/{{ cookiecutter.git_username }}/{{ cookiecutter.project_slug }}.git
+    $ git clone https://github.com/YOUR_USERNAME/{{ cookiecutter.project_slug }}.git
```

Comment:
I think the user name should ale be adapted with the cookie cutter.

Reply:
that should be write, its YOUR_USERNAME since we are talking about a forked repo to contribute

Reply:
Oh yeah, I missed that.